rationale/fees.md

```diff
@@ -6,6 +6,8 @@
 
 ## Preamble
 
+**This document will apply to a future version of the LazyLedger protocol.**
```

Comment:
What we could do: we could tag the current state of the spec (`v0.1.0-idealized`) and remove all things that we plan to add later. That might be clearer than this and we have a tagged version that we can refer to in future discussions?

Reply:
> (it can always be referenced by commit hash).

Sure, but I feel like adding a tag is easier to find and more explicit.